### PR TITLE
TST: add appveyor.yml

### PR DESCRIPTION
**Pull request by thequackdaddy**
Hello,

I put together a appveyor.yml if you would like to use it. 

This built successfully for me here:

https://ci.appveyor.com/project/thequackdaddy/blaze

A few caveats:
* mysql doesn't seem to work well on win32. I *think* the problem has something to do with loading data. (in most documentations that I've seen, mysql on win32 uses pathnames like `C:\mydir\myfile.csv` while I think the way the test suite and `odo` work is that they send pathnames like `C:\\mydir\\myfile.csv`. So I've skipped all those tests.
* mongodb fails a handful of tests. I've xfailed them in this version for now. 

Here's a sample mongo fail... 

```
bank = Collection(Database(MongoClient('localhost', 27017), 'test_db'), 'bank')
    def test_summary_arith_max(bank):
        expr = by(t.name, add_one_and_sum=(t.amount + 1).max())
        result = compute(expr, bank)
>       assert result == [('Bob', 301), ('Alice', 201)]
E       assert [('Alice', 201), ('Bob', 301)] == [('Bob', 301), ('Alice', 201)]
E         At index 0 diff: ('Alice', 201) != ('Bob', 301)
E         Full diff:
E         - [('Alice', 201), ('Bob', 301)]
E         + [('Bob', 301), ('Alice', 201)]
```

A more detailed fail is available here:

https://ci.appveyor.com/project/thequackdaddy/blaze/build/1.0.5

Ultimately (in a separate PR) I'd like to get mssql testing added which is why I'm submitting this. 

Thanks!

### CODE REVIEW

**Comment by necaris** [comment]
Could you push another commit so that the [project](https://ci.appveyor.com/project/blaze/blaze) starts building [on AppVeyor](https://ci.appveyor.com/project/blaze/blaze)?

**Comment by thequackdaddy** [comment]
@necaris re-pushed. all green, with the caveats above.

**Comment by necaris** [comment]
Thanks @thequackdaddy ! I'm happy to merge this as an overall improvement and a prelude to further work -- thoughts @llllllllll ?

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  - createdb test
+
+  # install the frozen ci dependencies
+  - pip install -e .[ci]
+
+  # blaze required deps
+  - pip install git+https://github.com/blaze/datashape
+  - pip install git+https://github.com/blaze/odo
+
+  - python setup.py develop
```
Comment (llllllllll) [inline appveyor.yml]: this is done by `pip install -e .`

File: blaze/compute/tests/test_mongo_compute.py [hunk 2]
```diff
@@ -268,30 +269,35 @@ def test_datetime_handling(events):
                        events)) == set([2, 3])
 
 
+@xfail(sys.platform == 'win32', reason='on win32, results are backwards')
```
Comment (llllllllll) [inline blaze/compute/tests/test_mongo_compute.py]: we can odo this to a set so order doesn't matter. That can be another change though,

**Comment by llllllllll** [comment]
Amazing! Thanks for adding this!

**Comment by thequackdaddy** [comment]
Made the change you requested. 

Looks like dask 0.13 (released today) broke some things. 

@mrocklin do you have any ideas ?

**Comment by llllllllll** [comment]
The `[ci]` dependencies should be frozen so that we can update this when needed.

**Comment by postelrich** [comment]
Closing to cleanup old PRs. Feel free to submit a new PR.

**Pull request closed without merging**